scripts/index.js

```diff
@@ -0,0 +1,53 @@
+var blog =
+$('article:first').clone()[0];
+
+article.prototype.toHTML = function (argument) {
+  $artSkel = $('article').filter(':first').clone();
+};
+
+function postArticle (rawArticle) {
+  var art = new ARticle(rawArticle);
+  art.toHTML();
+};
+
+var Article = function (props) {
+  this.author = props.authoer;
+  this.whatever = props.whatever;
+  articles.push(this);
+};
+
+Article.prototype.toHTML = function() {
+  var $clonedARticle = $('article#template').clone();
+  $clonedArticle.removeAttr('id');
+  $clonedArticle.find('h1.author').html('<a href "' + articles[i].authorURL + '" ' + articles[i].author);
+};
+
+blog.sortRawData = function () {
+  blog.RawData;
+};
+
+blog.truncateArticles = function() {
+  $('article p:not(:first-child')).hide();
```

Comment:
```javascript
$('article p:not(:first-child')).hide();
```
This still seems to work, but it's got an error. That closing `'` needs to be outside the (), so `$('article p:not(:first-child)').hide();`